frontend/src/tabs/authorship.pug

```diff
@@ -1,6 +1,7 @@
 #authorship
-  span.large-font Code Panel
-  .toolbar--multiline(v-if="hasCommits(info)")
```

Comment:
![Untitled](https://user-images.githubusercontent.com/3306138/81703727-8dec3580-949f-11ea-9e8f-b52a2c67c7de.png)

regression from the merge conflict

Reply:
Aah. Seemed to have missed that. Thanks for pointing it out.

Reply:
Fixed.